tests/data/gloframesstream/read.me

```diff
@@ -0,0 +1,12 @@
+The folder contains files are used for GLO nav message decoding unit tests.
```

Comment:
The folder contains files WHICH are used for GLO nav message decoding unit tests.